test/GithubApi.Repositories.test.js

```diff
@@ -0,0 +1,71 @@
+const { StatusCodes } = require('http-status-codes');
+const { expect } = require('chai');
+const chaiSubset = require('chai-subset');
+const chai = require('chai');
+const md5 = require('md5');
+
+chai.use(chaiSubset);
+const axios = require('axios');
+
+describe('Github Api Test', () => {
+  describe('Authentication', () => {
+    it('Via OAuth2 Tokens by Header', async () => {
+      const response = await axios.get('https://api.github.com/users/aperdomob', {
+        headers: {
+          Authorization: `token ${process.env.ACCESS_TOKEN}`
+        }
+      });
+      expect(response.status).to.equal(StatusCodes.OK);
+    });
+    it('Via OAuth2 Tokens by parameter', async () => {
+      const response = await axios.get(
+        'https://api.github.com/users/aperdomob',
+        { access_token: process.env.ACCESS_TOKEN }
+      );
+      expect(response.status).to.equal(StatusCodes.OK);
+    });
+  });
+});
```

Comment:
This is not required for this file.

Reply:
I delete this part and did the test again in github and do not works. In the last commit all test pass.